src/scenes/Home/TerrawareHomeView/CTACard.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+
+import { Box, SxProps, Typography, useTheme } from '@mui/material';
+import { Props as ButtonProps } from '@terraware/web-components/components/Button/Button';
+import { useDeviceInfo } from '@terraware/web-components/utils';
+
+import Button from 'src/components/common/button/Button';
+
+type CTACardProps = {
+  buttonsContainerSx?: SxProps;
+  description: string | (string | JSX.Element)[];
+  imageSource?: string;
+  padding?: number | string;
+  primaryButtonProps?: ButtonProps;
+  secondaryButtonProps?: ButtonProps;
+  title?: string | (string | JSX.Element)[];
+};
+
+const CTACard = ({
+  buttonsContainerSx,
+  description,
+  imageSource,
+  padding = '24px',
+  primaryButtonProps,
+  secondaryButtonProps,
+  title,
+}: CTACardProps): JSX.Element => {
+  const { isDesktop, isMobile, isTablet } = useDeviceInfo();
+  const theme = useTheme();
+
+  return (
+    <Box
+      sx={{
+        alignItems: 'center',
+        background: theme.palette.TwClrBg,
+        borderRadius: '8px',
+        display: 'flex',
+        flexDirection: isDesktop ? 'row' : 'column',
+        height: '100%',
+        justifyContent: 'space-between',
+        padding,
+      }}
+    >
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: isMobile ? 'column' : 'row',
+        }}
+      >
+        {imageSource && (
+          <Box
+            sx={{
+              marginBottom: isMobile ? '32px' : 0,
+              marginRight: isMobile ? 0 : '32px',
+              textAlign: 'center',
+            }}
+          >
+            <img src={imageSource} />
```

Comment:
I think we should add an `alt` attribute

Reply:
I agree 100%